CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change log
 
+## [v0.7.2] - unreleased
+* Expand supported rouge versions
```

Comment:
Can you please insert a 'Changed' header and keep an empty line before? (Please see previous entries for how it should look.) Can you also please modify to say 'Change gemspec to expand...' and add your full name with GitHub ref in brackets at the end?

Reply:
Done (I hope I did it right? 😅 )